src/formats/MachO.cc

```diff
@@ -2,8 +2,10 @@
 #include <QFile>
 
 #include <cmath>
+#include <iterator>
```

Comment:
Why is this added exactly?

Reply:
I think it was a mistake automatic import. Removed.